### PR TITLE
Increase wait time  in MetricsCacheSinkTest and TMasterSinkTest

### PR DESCRIPTION
**Pull request by nwangtw**
Restart wait time is changed to 3 times of reconnect time from 2. One reconnect
time at beginning and another delay of reconnect time caused by exception handler and extra time.

### CODE REVIEW

File: heron/metricsmgr/tests/java/com/twitter/heron/metricsmgr/sink/metricscache/MetricsCacheSinkTest.java [hunk 1]
```diff
@@ -118,9 +120,9 @@ public void testMetricsCacheClientService() throws Exception {
 
     // Then we check whether the MetricsCacheService has restarted the MetricsCacheClient for
     // several times Take other factors into account, we would check whether the MetricsCacheClient
-    // has restarted at least half the RESTART_WAIT_INTERVAL_SECONDS/RECONNECT_INTERVAL
+    // at least RESTART_WAIT_INTERVAL - RECONNECT_INTERVAL * 2
     assertTrue(metricsCacheSink.getMetricsCacheStartedAttempts()
-        >= (RESTART_WAIT_INTERVAL.getSeconds() / RECONNECT_INTERVAL.getSeconds() / 2));
+        >= (RESTART_WAIT_INTERVAL.getSeconds() - 2 * RECONNECT_INTERVAL.getSeconds()));
```
Comment (Yaliang) [inline heron/metricsmgr/tests/java/com/twitter/heron/metricsmgr/sink/metricscache/MetricsCacheSinkTest.java]: Isn't `metricsCacheStartedAttempts` the number of attempts?

Comment (nwangtw) [reply heron/metricsmgr/tests/java/com/twitter/heron/metricsmgr/sink/metricscache/MetricsCacheSinkTest.java]: Yeah. It is the number of attempts to connect to a specific endpoint.

Comment (nwangtw) [reply heron/metricsmgr/tests/java/com/twitter/heron/metricsmgr/sink/metricscache/MetricsCacheSinkTest.java]: In fact, let me think about this line again. The idea for the old and new code are similar and the new code focus on the first time. I may consider reverting this line. Thanks for the question.

Comment (Yaliang) [reply heron/metricsmgr/tests/java/com/twitter/heron/metricsmgr/sink/metricscache/MetricsCacheSinkTest.java]: Yeah, I was feeling it's wired to compare (x times >= y seconds).

File: heron/metricsmgr/tests/java/com/twitter/heron/metricsmgr/sink/metricscache/MetricsCacheSinkTest.java [hunk 2]
```diff
@@ -48,7 +48,10 @@
       TopologyMaster.MetricsCacheLocation.newBuilder().getDescriptorForType().getFullName();
 
   private static final Duration RECONNECT_INTERVAL = Duration.ofSeconds(1);
-  private static final Duration RESTART_WAIT_INTERVAL = Duration.ofSeconds(2);
+  // Restart wait time is set at 2 times of reconnect time plus another second. The 2 times factor
+  // is because of localtion checking event interval and the sleep of reconnect interval in
```
Comment (Yaliang) [inline heron/metricsmgr/tests/java/com/twitter/heron/metricsmgr/sink/metricscache/MetricsCacheSinkTest.java]: typo: localtion -> location

File: heron/metricsmgr/tests/java/com/twitter/heron/metricsmgr/sink/tmaster/TMasterSinkTest.java [hunk 3]
```diff
@@ -48,7 +48,10 @@
       TopologyMaster.TMasterLocation.newBuilder().getDescriptorForType().getFullName();
 
   private static final Duration RECONNECT_INTERVAL = Duration.ofSeconds(1);
-  private static final Duration RESTART_WAIT_INTERVAL = Duration.ofSeconds(2);
+  // Restart wait time is set at 2 times of reconnect time plus another second. The 2 times factor
+  // is because of localtion checking event interval and the sleep of reconnect interval in
```
Comment (Yaliang) [inline heron/metricsmgr/tests/java/com/twitter/heron/metricsmgr/sink/tmaster/TMasterSinkTest.java]: typo: localtion -> location

File: heron/metricsmgr/tests/java/com/twitter/heron/metricsmgr/sink/tmaster/TMasterSinkTest.java [hunk 4]
```diff
@@ -117,7 +120,7 @@ public void testTMasterClientService() throws InterruptedException {
 
     // Then we check whether the TMasterService has restarted the TMasterClient for several times
     // Take other factors into account, we would check whether the TMasterClient has restarted
-    // at least half the RESTART_WAIT_INTERVAL/RECONNECT_INTERVAL
+    // at least half the RESTART_WAIT_INTERVAL_SECONDS/RECONNECT_INTERVAL
```
Comment (Yaliang) [inline heron/metricsmgr/tests/java/com/twitter/heron/metricsmgr/sink/tmaster/TMasterSinkTest.java]: `RESTART_WAIT_INTERVAL/RECONNECT_INTERVAL`?

**Pull request merged**